lib/codecov.js

```diff
@@ -4,21 +4,21 @@ var request = require('teeny-request').teenyRequest
 var urlgrey = require('urlgrey')
 var jsYaml = require('js-yaml')
 var walk = require('ignore-walk')
+var execFileSync = require('child_process').execFileSync
 var execSync = require('child_process').execSync
 
 var detectProvider = require('./detect')
 
 var version = 'v' + require('../package.json').version
 
-var patterns,
-  more_patterns = ''
+var patterns, more_patterns, winPatterns
```

Comment:
```suggestion
var patterns, more_patterns, winPatterns = ''
```

The ` = ''` is required to prevent the undefined error we're seeing.